src/ControlSystem/TimescaleTuner.cpp

```diff
@@ -33,12 +34,12 @@ struct TimescaleCreator {
 };
 }  // namespace
 
-TimescaleTuner::TimescaleTuner(
+template <bool AllowDecrease>
+TimescaleTuner<AllowDecrease>::TimescaleTuner(
     const typename InitialTimescales::type& initial_timescale,
     const double max_timescale, const double min_timescale,
-    const double decrease_timescale_threshold,
     const double increase_timescale_threshold, const double increase_factor,
-    const double decrease_factor)
+    const double decrease_timescale_threshold, const double decrease_factor)
```

Comment:
Do some sort of check that these are reasonable in the no-decrease case.  Check `decrease_factor == 1.0`, at least.

Reply:
Only did the `decrease_factor == 1.0` check because I pass `limits::max()` for the decrease threshold and that's less intuitive of a default.